packages/terra-arrange/src/Arrange.jsx

```diff
@@ -63,10 +63,6 @@ const Arrange = ({
   fitEndAttributes,
   ...customProps
   }) => {
-  if (fitStart === undefined && fitEnd === undefined) {
```

Comment:
Removed at @bjankord's instruction, as there exist use-cases in which we shouldn't **force** the user to provide a `fitStart` or `fitEnd`.

cc: @bjankord, @dm123455, @neilpfeiffer, @alex-bezek.